src/ui/map.js

```diff
@@ -2652,9 +2654,12 @@ class Map extends Camera {
      * @returns {Map} Returns itself to allow for method chaining.
      * @example
      * map.setFog({
-     *     "range": [1.0, 12.0],
-     *     "color": 'white',
-     *     "horizon-blend": 0.1
+     *     "range": [0.8, 8],
+     *     "color": "#dc9f9f",
+     *     "horizon-blend": 0.5,
+     *     "high-color": "#245bde",
+     *     "space-color": "#000000",
+     *     "star-intensity": 0.15
```

Comment:
Nice fog 👍 

![image](https://user-images.githubusercontent.com/14878684/173958543-34f9d5cc-4c7a-4a07-90c7-c4ff188d4101.png)

Reply:
It's one of the presets from the atmosphere playground!